service-deployment/src/gpu-exporter/dockerfile.template

```diff
@@ -17,22 +17,29 @@
 
 FROM ubuntu:16.04
 
+#
+# Preparation
+#
+
 ENV NVIDIA_VERSION=current
 ENV NV_DRIVER=/var/drivers/nvidia/$NVIDIA_VERSION
 ENV LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$NV_DRIVER/lib:$NV_DRIVER/lib64
 ENV PATH=$PATH:$NV_DRIVER/bin
 
+WORKDIR /root/
+
 RUN apt-get update && \
-    apt-get install -y wget && \
-    apt-get -y install golang --no-install-recommends && \
-    rm -r /var/lib/apt/lists/*
+    apt-get -y install wget build-essential python python-pip git
 
-WORKDIR /go
+COPY copied_file/exporter/* /usr/local/
 
-COPY copied_file/exporter/gpu_exporter.go /usr/local/
-COPY start.sh /usr/local/start.sh
-RUN go build -v -o /usr/local/gpu_exporter /usr/local/gpu_exporter.go
-RUN chmod u+x /usr/local/start.sh
-RUN chmod u+x /usr/local/gpu_exporter 
-CMD [ "/usr/local/start.sh" ]
+RUN wget https://download.docker.com/linux/static/stable/x86_64/docker-17.06.2-ce.tgz
+RUN cp docker-17.06.2-ce.tgz /usr/local
+RUN tar xzvf /usr/local/docker-17.06.2-ce.tgz -C /usr/local/
+RUN cp -r /usr/local/docker/* /usr/bin/
+
+#
+# start
+#
 
+CMD python /usr/local/job_exporter.py {{ clusterconfig['clusterinfo'][ 'dataPath' ] }}/prometheus 30
```

Comment:
Remove this and hardcode a path. And hardcord the path of container's fs in the kubernetes yaml file. Don't use any cluster specific configuration when building image